### PR TITLE
Show default persona icon when viewing someone else's chat session

### PR DESCRIPTION
**Pull request by spolu**

### CODE REVIEW

File: front/pages/w/[wId]/u/chat/[cId]/index.tsx [hunk 1]
```diff
@@ -411,11 +414,18 @@ export function MessageView({
...
-                <img
-                  className="h-8 w-8 rounded-md"
-                  src={user?.image || "https://gravatar.com/avatar/anonymous"}
-                  alt=""
-                />
+                {!readOnly && user?.image ? (
+                  <img
+                    className="h-8 w-8 rounded-md"
+                    src={
+                      user?.image ||
```
Comment (philipperolet) [inline front/pages/w/[wId]/u/chat/[cId]/index.tsx:421]: here user?.image always true (checked above)

File: front/pages/w/[wId]/u/chat/[cId]/index.tsx [hunk 2]
```diff
@@ -411,11 +414,18 @@ export function MessageView({
...
+                      "https://gravatar.com/avatar/anonymous?d=mp"
+                    }
+                    alt=""
+                  />
+                ) : (
+                  <UserCircleIcon className="mx-1 my-1 h-6 w-6 text-gray-300"></UserCircleIcon>
```
Comment (philipperolet) [inline front/pages/w/[wId]/u/chat/[cId]/index.tsx:427]: no case in which gravatar url will show anymore -- just checking it's indeed the intended behavior

**Review by philipperolet — COMMENTED** [review]
1 com to check otherwise lgtm

File: front/pages/w/[wId]/u/chat/[cId]/index.tsx (hunk 1 above)
Comment (spolu) [reply front/pages/w/[wId]/u/chat/[cId]/index.tsx:421]: You're right

File: front/pages/w/[wId]/u/chat/[cId]/index.tsx (hunk 2 above)
Comment (spolu) [reply front/pages/w/[wId]/u/chat/[cId]/index.tsx:427]: Will fix

Comment (spolu) [reply front/pages/w/[wId]/u/chat/[cId]/index.tsx:424]: (done)

**Comment by spolu** [comment]
(merging post-fix-merge review welcome)

**Comment by philipperolet** [comment]
:+1:

**Pull request merged**